packages/core-api/src/versions/2/delegates/schema.ts

```diff
@@ -52,9 +52,19 @@ export const show: object = {
 };
 
 export const search: object = {
-    query: pagination,
+    query: {
+        ...pagination,
+        ...{
+            orderBy: Joi.string(),
+        },
+    },
     payload: {
         username: schemaUsername,
+        usernames: Joi.array()
+            .unique()
+            .min(1)
+            .max(50)
```

Comment:
Maybe instead of hard coding this use the number of active delegates https://github.com/ArkEcosystem/core/blob/master/packages/crypto/src/networks/mainnet/milestones.json#L5

Reply:
okay like that?